test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGenJsonOptionsTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Swashbuckle.AspNetCore.SwaggerGen.DependencyInjection;
+using Swashbuckle.AspNetCore.SwaggerGen.Test.Fixtures;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen.Test;
+
+public class SwaggerGenJsonOptionsTests
```

Comment:
Could we add a test here that checks for the `JsonSerializerOptions.Default` case please? We might have one already elsewhere, which if that's the case it can be moved here instead.